src/components/Grants/CliffNotice.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import Time from 'decentraland-gatsby/dist/utils/date/Time'
+
+import { CLIFF_PERIOD_IN_DAYS } from '../../entities/Proposal/utils'
+import DateTooltip from '../Common/DateTooltip'
+import InfoCircle from '../Icon/InfoCircle'
+
+import './CliffNotice.css'
+
+interface Props {
+  vesting_start_date: number
```

Comment:
I'd rename this to `vestingStartDate` as we always use camel case in the frontend and snake case in variables coming from the back.